### PR TITLE
'OpenShift: New Application' command to command palette

### PR DESCRIPTION
**Pull request by sudhirverma**
Issue: #230

### CODE REVIEW

**Comment by sudhirverma** [comment]
testPR

**Comment by sudhirverma** [comment]
testPR

File: src/openshift/application.ts [hunk 1]
```diff
@@ -10,17 +10,33 @@ import * as validator from 'validator';
...
+            if (projectName.length === 0) {
+                await vscode.window.showErrorMessage('Sorry there are no project to create an application (Please create new project and try again)');
+            }else if (projectName.length > 1 ) {
+                projectName = await vscode.window.showQuickPick(projectName, {placeHolder: "In which project you want to create Application"});
+            } else if (projectName[0]) {
+                projectName = projectName[0];
             }
-        });
+        }
+        if (!(projectName.length === 0)) {
```
Comment (dgolovin) [inline src/openshift/application.ts]: simpler `projectName.length > 0`

File: src/openshift/application.ts [hunk 2]
```diff
@@ -10,20 +10,36 @@ import * as validator from 'validator';
...
-        const appName = await vscode.window.showInputBox({
-            prompt: "Application name",
-            validateInput: (value: string) => {
-                if (validator.isEmpty(value.trim())) {
-                    return 'Empty application name';
-                }
-                if (!validator.matches(value.trim(), '^[a-z0-9]([-a-z0-9]*[a-z0-9])*$')) {
-                    return 'Not a valid application name. Please use lower case alphanumeric characters or "-", and must start and end with an alphanumeric character';
-                }
+        let projectObj;
```
Comment (dgolovin) [inline src/openshift/application.ts]: Please do not use the local variables to store object of different types.

File: src/openshift/application.ts [hunk 3]
```diff
@@ -10,20 +10,36 @@ import * as validator from 'validator';
...
+            if (projectObj.length === 0) {
+                await vscode.window.showErrorMessage('Sorry there are no project to create an application (Please create new project and try again)');
+            }else if (projectObj.length > 1 ) {
+                projectObj = await vscode.window.showQuickPick(projectObj, {placeHolder: "In which project you want to create Application"});
+            } else if (projectObj[0]) {
+                projectObj = projectObj[0];
             }
-        });
+        }
+        if (!(projectObj.length === 0)) {
```
Comment (dgolovin) [inline src/openshift/application.ts]: should be just 
``` javascript
if (projectObj) {
```
considering you not using `projectObj` just to store selected oproject

**Comment by sudhirverma** [comment]
@dgolovin please review this pr

**Pull request merged**